### ISSUE TITLE
"regular resource usage tracking for 100 pods per node over 20m0s" panic

### ISSUE THREAD

**Issue by spxtr** [kubernetes/kubernetes#24214]
```
STEP: deleting replication controller resource300-2b09893b-01a3-11e6-b712-42010af01555 in namespace e2e-tests-kubelet-perf-agbbv
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4fa356]

goroutine 29580 [running]:
panic(0x1bc8d40, 0xc820016110)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
k8s.io/kubernetes/test/e2e.getSystemContainerStats(0x0, 0xc820ba63f0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_stats.go:314 +0x226
k8s.io/kubernetes/test/e2e.(*resourceCollector).collectStats(0xc8217e0ba0, 0xc8214cf5f0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_stats.go:504 +0x76
k8s.io/kubernetes/test/e2e.(*resourceCollector).Start.func1()
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_stats.go:492 +0x29
k8s.io/kubernetes/pkg/util/wait.JitterUntil.func1(0xc8217d2820)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:66 +0x4b
k8s.io/kubernetes/pkg/util/wait.JitterUntil(0xc8217d2820, 0x2540be400, 0x0, 0xc8217e0c00)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:67 +0x68
k8s.io/kubernetes/pkg/util/wait.Until(0xc8217d2820, 0x2540be400, 0xc8217e0c00)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:47 +0x3e
created by k8s.io/kubernetes/test/e2e.(*resourceCollector).Start
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_stats.go:492 +0x111
```
Happening in both master gce and gke serial suites as of about noon yesterday. [Example](https://console.cloud.google.com/storage/browser/kubernetes-jenkins/logs/kubernetes-e2e-gce-serial/1060).

**Comment by yujuhong** [comment]
Fix has already been merged this morning #24205. 

It just got picked up by the current build. If it passes, I'll close the issue.

**Comment by yujuhong** [comment]
The suite is still running but the test in question has already passed. Closing the issue.
